azavea/pwd-unitybar
# Add mapActionTitle and helpActionTitle props

## Overview

Enable passing custom mapActionTitle & helpActionTitle props from host
application.

Update demo & README.

Connects #78 
Connects https://github.com/azavea/pwd-parcel-viewer/issues/719

### Demo

![Screen Shot 2019-04-19 at 3 20 25 PM](https://user-images.githubusercontent.com/4165523/56440182-ad258d00-62b6-11e9-9606-7758642ee12e.png)


## Testing Instructions

- visit the Netlify deployment demo page and verify that you can change the button title for the map action and help action by updating their respective prop
